drivers/hmis/app/graphql/types/hmis_schema/service_input.rb

```diff
@@ -20,9 +20,10 @@ class HmisSchema::ServiceInput < Types::BaseInputObject
     argument :referral_outcome, HmisSchema::Enums::Hud::PATHReferralOutcome, required: false
 
     def to_params
-      result = to_h
+      result = to_h.except(:service_type)
 
-      result[:type_provided] = type_provided.split(':').last&.to_i if type_provided.present?
+      result[:record_type] = record_type || type_provided&.split(':')&.first&.to_i
+      result[:type_provided] = type_provided&.split(':')&.last&.to_i
       result[:sub_type_provided] = sub_type_provided.split(':').last&.to_i if sub_type_provided.present?
```

Comment:
Hmm I'm not following. I think these selected lines should all be removed.. These inputs are enums, like `SSVF_SERVICE__ASSISTANCE_OBTAINING_VA_BENEFITS` –– they cannot be split on `:`.